backend-api/template.yaml

```diff
@@ -259,6 +259,13 @@ Resources:
       Environment:
         Variables:
           CLIENT_REGISTRY_SECRET_NAME: !Sub ${Environment}/clientRegistry
+      VpcConfig:
+        SubnetIds:
+          - !ImportValue devplatform-vpc-PrivateSubnetIdA
+          - !ImportValue devplatform-vpc-PrivateSubnetIdB
+          - !ImportValue devplatform-vpc-PrivateSubnetIdC
+        SecurityGroupIds:
+          - !ImportValue devplatform-vpc-AWSServicesEndpointSecurityGroupId
```

Comment:
If we need to create our own security group instead of referencing this one (i.e. `AWSServicesEndpointSecurityGroupId`), can we perhaps rename the `ProxyLambdaSecurityGroup` to something like `LambdaEgressSecurityGroup` and reference this security group throughout this template instead of `AWSServicesEndpointSecurityGroupId`? This existing security group seems to be suitable. cc @sandymay-dd